### PR TITLE
Add Drawer 'before' and 'after' positions value

### PR DESCRIPTION
**Pull request by Yarovaya**

### CODE REVIEW

File: js/ui/drawer/ui.drawer.js [hunk 1]
```diff
@@ -325,6 +325,17 @@ const Drawer = Widget.inherit({
         this._minSize = this.option("minSize") || 0;
     },
 
+    getDrawerPosition() {
+        let position = this.option("position");
```
Comment (DokaRus) [inline js/ui/drawer/ui.drawer.js]: Use `const` instead of `let`

File: js/ui/drawer/ui.drawer.js [hunk 2]
```diff
@@ -370,7 +381,9 @@ const Drawer = Widget.inherit({
     },
 
     isHorizontalDirection() {
-        return this.option("position") === "left" || this.option("position") === "right";
+        var position = this.getDrawerPosition();
```
Comment (DokaRus) [inline js/ui/drawer/ui.drawer.js]: `const`?

File: js/docEnums.js [hunk 3]
```diff
@@ -627,7 +627,7 @@
 
 /**
  * @typedef {string} Enums.DrawerPosition
- * @enum {'left'|'right'|'top'|'bottom'}
+ * @enum {'left'|'right'|'top'|'bottom'|'auto'}
```
Comment (DokaRus) [inline js/docEnums.js]: I think we should have a couple of new values. 
One of it setts Drawer's panel default position by the left side for a LTR locale, and by the right side for a RTL locale(`before`? `start`?).
Another setts Drawer's panel position by the right side for a LTR locale, and by the left side for a RTL locale (`after`? `end`?).

Comment (DokaRus) [reply js/docEnums.js]: Take a look to the experimental [text-align](https://developer.mozilla.org/en-US/docs/Web/CSS/text-align#Values) values.

Comment (viterobk) [reply js/docEnums.js]: How popular are RTL scenarios? If they are not so popular, in most cases, users can be confused by duplicated values ("left"|"before" & "right"|"after"). And the case when the menu is displayed at the right for LTR and left for RTL looks rare. However, if someone needs to configure Drawer this way, he or she can set this option to "left" or "right" depending on the LTR/RTL mode.

Comment (DokaRus) [reply js/docEnums.js]: >How popular are RTL scenarios?

We have feedback related to the Drawer usage with RTL right now

>users can be confused by duplicated values

`left` or `right` are **static** positions. `before` and `after` - adaptive. Use adaptive values for a common scenarios, or use static value to set Drawer's panel position for the any locale.

>However, if someone needs to configure Drawer this way, he or she can set this option to "left" or "right" depending on the LTR/RTL mode.

So, you suggest to drop RTL support for this widget?

Comment (viterobk) [reply js/docEnums.js]: I suggest to support RTL by default. If position is "auto", the menu is displayed at the left for LTR and at the right for RTL. However, if users ask for adaptive values, "before" and "after" look good.

**Review by DokaRus — APPROVED** [review]
LGTM

**Review by jsdmitry — APPROVED** [review]

**Pull request merged**